### ISSUE TITLE
error: cannot find symbol class Configuration

### ISSUE THREAD

**Issue by FridaySG** [parse-community/Parse-SDK-Android#961]
Been at this for hours guys, any help would be very much appreciated. The error I'm getting is: 
`error: cannot find symbol class Configuration. `

SDK version 1.2.0.

This was happening before I refactored to androidx, so I refactored hoping that it would solve the issue after I noticed androidx support in the change-log but that didn't solve it.

My targetSdk and compileSdk API version are both 28, although I've also tried 29 with no success.

```
import com.parse.Parse;
import com.parse.facebook.ParseFacebookUtils;
import com.parse.ParseInstallation;

import io.fabric.sdk.android.Fabric;

public class AppClass extends Application {

    public void onCreate() {
        super.onCreate();
        Fabric.with(this, new Crashlytics());

        new AsyncTask<Void, Void, Void>() {
            @Override
            protected Void doInBackground(Void... params) {
                return null;
            }
        };

        Parse.initialize(new Parse.Configuration.Builder(this)
                .applicationId("my-app-id")
                .clientKey(null)
                .server("my-server")
                .enableLocalDataStore()
                .build()
        );
```

Gradle:

```
implementation 'com.github.parse-community.Parse-SDK-Android:parse:1.20.0'
implementation 'com.github.parse-community.Parse-SDK-Android:fcm:1.20.0'
implementation 'com.github.parse-community:ParseFacebookUtils-Android:1.14.0'
```
I also tried the basic stuff, invalidate caches, clean etc.

**Comment by FridaySG** [comment]
Finally figured it out. I accidentally had an old ParseUI library in my build.gradle that for some reason I was blind to. When I tried to build the project with the initialization block commented out, the compiler whined that I had conflicting classes (two different Parse libraries were being loaded) because the old ParseUI library (0.0.1) had Parse 1.10.3 as a dependency. Utter foolishness on my part. Updated the ParseUI library to the latest (0.0.5) and back to business as usual.
